### DISCUSSION TITLE
[Feature] Searching on shared albums; Manual thumbnails for video clips

### DISCUSSION THREAD

**Discussion by laurfb** [immich-app/immich#11473] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I bring to your attention a few feature request, maybe will help:
1. Searching on shared albums (now seems that we can only browse on shared albums)
2. The possibility to create manual thumbnails for video clips (manual select a custom frame from clip as thumbnail - using slider or setting direct the frame time code or even uploading one from local computer). 
3. Maybe changind "Photos" text option from main page (left zone menu) to "Media" or "Photos/Videos" since we can have in Immich both photos and videos?
regards,
Laur

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
As requested in the template, only one feature per post please.
